src/modules/Servicedomain/Api/Admin.php

```diff
@@ -211,6 +213,13 @@ public function tld_delete($data)
         if (!$model instanceof \Model_Tld) {
             throw new \Box_Exception('TLD not found');
         }
+        // check if tld is used by any domain
+        $service_domains = $this->di['db']->find('ServiceDomain', 'tld = ?', ['tld' => $model->tld]);
+        $count = count($service_domains);
+        if ($count > 0) {
+            throw new \Box_Exception('TLD is used by :count: domains', ['count' => $count], 707);
```

Comment:
You are using the placeholder wrong here. It needs to be `:count:` in both places, not just one